howto_docker.md

```diff
@@ -0,0 +1,58 @@
+# How to use Docker and Openfisca
+
+## Run OpenFisca on Docker
+
+1. Install Docker
+2. create a docker file in the directory you wish to work on.
+```
```

Comment:
Newline ? (maybe is not needed, but I've seen them applied consistently through the doc). This comment applies to the whole file.